data-products/_example_2/package_data_product.py

```diff
@@ -3,8 +3,8 @@
 
 
 def zip_data_product(folder: str):
-    with zipfile.ZipFile(f"data_product_{folder}.zip", "w") as f:
-        for file in glob.glob(f"./{folder}/application/*") + glob.glob(f"./{folder}/requirements.txt"):
+    with zipfile.ZipFile(f"data_product{folder}.zip", "w") as f:
```

Comment:
this is better, can you change the workflow which pushes the file so it's in line with this file naming

Reply:
I dont get what you mean by this ?

Reply:
https://github.com/ministryofjustice/data-platform-products/blob/a6850352d992aa52a6b186972d266d752752bd7b/.github/workflows/data-product-example-2-poc.yaml#L47